jobrunner/run.py

```diff
@@ -120,6 +120,11 @@ def handle_job(job, api, mode=None, paused=None):
     if paused:
         if job.state == State.PENDING:
             # do not start the job, keep it pending
+            set_state(
```

Comment:
I think you'd be better of using `set_message()` here because you're not actually changing the state , and that will avoid being so spammy in the logs:
https://github.com/opensafely-core/job-runner/blob/75d76821ac0aafc995ae51dfaed184cd5f0e52a6/jobrunner/run.py#L420